remill/Arch/AArch64/Arch.cpp

```diff
@@ -1774,6 +1783,148 @@ bool TryDecodeMOV_ORR_64_LOG_IMM(const InstData &data, Instruction &inst) {
   return true;
 }
 
+
+// CLZ  <Wd>, <Wn>
+bool TryDecodeCLZ_32_DP_1SRC(const InstData &data, Instruction &inst) {
+  AddRegOperand(inst, kActionWrite, kRegW, kUseAsAddress, data.Rd);
+  AddRegOperand(inst, kActionRead, kRegW, kUseAsAddress, data.Rn);
+  return true;
+}
+
+// CLZ  <Xd>, <Xn>
+bool TryDecodeCLZ_64_DP_1SRC(const InstData &data, Instruction &inst) {
+  AddRegOperand(inst, kActionWrite, kRegX, kUseAsAddress, data.Rd);
+  AddRegOperand(inst, kActionRead, kRegX, kUseAsAddress, data.Rn);
+  return true;
+}
+
+bool DecodeConditionalRegSelect(const InstData &data, Instruction &inst, RegClass r_class, bool second_source) {
```

Comment:
I suggest adding a simple `enum` with two entries, one that is like "two sources", the other as "one source", as opposed to `true` and `false`.